docs/BR.md

```diff
@@ -1008,6 +1042,83 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
+| ACME “tls-alpn-01” method for IP Addresses (defined in 3.2.2.5.7) | This method involves establishing a TLS connection.<br><br>
+“Corroborate” means the corresponding challenge information (e.g., Random Value or Request Token) MUST be observed by the required number of Network Perspectives defined in the "# of Network Perspectives and Quorum Requirements" table, otherwise the certificate MUST NOT be issued. |
+
+
+Table: Defining Corroborating CAA Evidence
+
+| CAA Method | Description of Corroborating Evidence |
+|---|---|
+| Any method | “Corroborate” means the required number of Network Perspectives defined in the "# of Network Perspectives and Quorum Requirements" table MUST have CAA lookup results that permit issuance for the domain in question. |
+
+Computing systems performing Multi-Perspective Domain Validation:
+
+- MUST:
+    - Forward all Internet traffic via a network or set of networks that filter all RPKI-invalid BGP routes as defined by RFC 6811 
+
+- SHOULD:
+    - Facility & Service Provider Requirements
+      - Be hosted from an ISO/IEC 27001 certified facility or equivalent. 
+      - Rely on services covered in one of the following reports: System and Organization Controls 3 (SOC 3), IASE 3000, ENISA 715, FedRAMP Moderate, C5:2020, CSA STAR CCM, or equivalent.
+    - Network Hardening
+      - Configure each network boundary control (firewall, switch, router, gateway, or other network control device or system) with rules that support only the services, protocols, ports, and communications identified as necessary to its operations.
+    - System Hardening
+      - Disable all accounts, applications, services, protocols, and ports that are not used.
+      - Implement multi-factor authentication for all user accounts.
+    - Vulnerability Detection and Patch Management
+      - Implement intrusion detection and prevention controls to protect against common network and system threats.
+      - Document and follow a vulnerability correction process that addresses the identification, review, response, and remediation of vulnerabilities.
+      - Undergo or perform a Vulnerability Scan at least every 3 months.
+      - Undergo a Penetration Test on at least an annual basis
+      - Apply recommended security patches within six (6) months of the security patch’s availability, unless the CA documents that the security patch would introduce additional vulnerabilities or instabilities that outweigh the benefits of applying the security patch.
+
+Beyond the above considerations, computing systems performing Multi-Perspective Domain Validation are considered outside of the audit scope described in Section 8 of this Policy.
```

Comment:
Also, to Aaron's [point](https://github.com/ryancdickson/staging/pull/6/files#r1224913856), we should replace "computing systems performing Multi-Perspective Domain Validation" with just "Network Perspectives".

If the "computing systems" element needs to be included in the "Network Perspective" concept, we should update the definition.

Reply:
I am fine with the change here from "computing systems performing Multi-Perspective Domain Validation" to "Network Perspective," but I think some tweaking of the definition of "Network Perspective" is needed. Specifically a "Network Perspective" is defined as "A configuration for sending outbound Internet traffic..." I think it is somewhat strange to thing about these requirements like a penetration test being enforced on a configuration. Maybe to resolve this we could update the first line of the "Network Perspective" definition to read "A system for sending outbound Internet traffic..."